### ISSUE TITLE
PowerSchool "what if"

### ISSUE THREAD

**Issue by ThatXliner** [ThatXliner/skootils#44]
### Prerequisites

- [X] I have searched the [issues](https://github.com/ThatXliner/powerschool/issues) and believe that it has not already been requested (and possibly rejected)
- [X] I have made sure that this feature isn't possible with this project's plugin system and/or configuration and core changes must be made to make this happen
- [X] I agree to follow the [Code of Conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/)

### Feature description

Implement PowerSchool "what if"

**Comment by ThatXliner** [comment]
I've been told that maybe the weights are in the syllabus. I'll need to check (because that's definitely a better way than this). But I don't have access to learn@vcs so this is blocked

**Comment by ThatXliner** [comment]
Unblocked due to discovery https://github.com/ThatXliner/skootils/issues/27#issuecomment-1176300833

**Comment by ThatXliner** [comment]
Another discovery: the scraped `weight`s don't matter (it's always 1). Is this always unweighted? I'm going to add a "beta" label on it lol..

**Comment by ThatXliner** [comment]
There might be weights because `_assignmentcategoryassociations > _teachercategory > color` but some shallow testing leads me to think that those don't matter. Quizzes are more than tests, according to colors. And I can't find a equation that works.

But just because I can't find one doesn't mean there isn't. Further investigation will be needed after beta.

**Comment by ThatXliner** [comment]
There are weights. In conservatory classes, etc. Uh new API output as well...

**Comment by ThatXliner** [comment]
On the weights, I can't find any mentioning of them from API output. I guess the user will have to manually add them. I'll add that as a separate issue
